oas_docs/overlays/kibana.overlays.yaml

```diff
@@ -14,6 +15,60 @@ actions:
         variables:
           kibana_url:
             default: localhost:5601
+  # Remove operation-level security definitions
+  - target: "$.paths['/api/status']['get'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/encrypted_saved_objects/_rotate_key']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_bulk_create']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_bulk_delete']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_bulk_get']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_bulk_resolve']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_bulk_update']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_export']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_find']['get'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_import']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/_resolve_import_errors']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/{type}']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/{type}/{id}']['get'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/{type}/{id}']['post'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/{type}/{id}']['put'].security"
+    description: Remove system security definitions
+    remove: true
+  - target: "$.paths['/api/saved_objects/resolve/{type}/{id}']['get'].security"
+    description: Remove system security definitions
+    remove: true
+  # Remove elastic-api-version parameter
+  - target: "$.paths['/api/status']['get'].parameters[?(@.name=='elastic-api-version')]"
```

Comment:
>   - target: "$.paths['/api/status']['get'].parameters[?(@.name=='elastic-api-version')]"

@jloleysens I added this overlay since I was surprised to see it in the non-serverless file. However, if it's valid in both contexts, I'll omit this overlay action.